packages/vm/README.md

```diff
@@ -140,18 +140,13 @@ const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Berlin })
 const vm = new VM({ common })
 ```
 
-If your use case only supports a certain range of `hardforks`, you can use the `supportedHardforks` option to restrict hardfork access on the `VM` instance:
+Please note that the `VM` constructor will throw an error if you pass in a `Common` instance that is set to a hardfork that the `VM` does not currently support. For example, the following would throw because the `VM` does not yet support the Merge hardfork:
 
 ```typescript
-const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Byzantium })
-const vm = new VM({
-  common,
-  supportedHardforks: [Hardfork.Byzantium, Hardfork.Constantinople],
-})
+const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Merge })
+const vm = new VM({ common })
```

Comment:
Actually I would remove this whole doc section. We have already the list with supported HFs above, it doesn't provide much value to describe that things won't work when a non-supported HF is used.

This section is also coming very much from the old not-all-HFs-backported days. 🙂